scripts/pages.mjs

```diff
@@ -43,6 +43,31 @@ class ClassPages extends Application {
     html[0].querySelectorAll(".class-nav img[data-action]").forEach(n => {
       n.addEventListener("click", this._onClickClass.bind(this));
     });
+    html[0].querySelector(".class-nav .gallery").addEventListener(
+      "wheel", debounce((event) => this._onGalleryScroll(event, html[0].querySelector(".class-nav .gallery")), 50)
+    );
```

Comment:
It should be possible to simply `bind` the function, and shouldn't need to use the same selector twice since `event.currentTarget` will refer to the element that the listener is targeting. We'll also want to use `foundry.utils.debounce` despite the longer name.

Reply:
Got it working, but cannot make it aesthetically pleasing.